docset/windows/dism/Add-AppxProvisionedPackage.md

```diff
@@ -236,6 +236,23 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -Regions
+Specifies what regions an app package (.appx or .appxbundle) must be provisioned in. The region argument can either be “all”, indicating that the app should be provisioned for all regions, or it can be a semi-colon delimited list of regions. The regions will be in the form of [ISO 3166-1 Alpha-2 or ISO 3166-1 Alpha-3 codes](https://en.wikipedia.org/wiki/ISO_3166-1). For example, the United States can be specified as either "US" or "USA" (case-insensitive). When a list of regions is not specified, the package will be provisioned only if it is pinned to start layout.
```

Comment:
Are we allowed to put a link to wikipedia? just wondering... Idk

Reply:
I took it from here so I guess yes...
https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/dism-app-package--appx-or-appxbundle--servicing-command-line-options#:~:text=an%20app%20package%20(.-,appx%20or%20.,%2D1%20Alpha%2D3%20codes.

Reply:
ok, thanks!